modules/ml/src/svm.cpp

```diff
@@ -2261,6 +2261,17 @@ Ptr<SVM> SVM::create()
     return makePtr<SVMImpl>();
 }
 
+Ptr<SVM> SVM::load(const String& filename)
+{
+    FileStorage fs;
+    fs.open(filename, FileStorage::READ);
+
+    Ptr<SVM> svm = makePtr<SVMImpl>();
+
+    svm->read(fs.getFirstTopLevelNode());
```

Comment:
Try to use here something like this `((SVMImpl*)svm.get())->read(...)` and remove virtual `read` method (ABI reports about vtable modification)